packages/p2p-v2/src/components/Modals/PaymentMethods/PaymentMethodErrorModal/PaymentMethodErrorModal.scss

```diff
@@ -1,27 +1,32 @@
 .p2p-v2-payment-method-error-modal {
-    &__wrapper {
-        display: flex;
-        flex-direction: column;
-        justify-content: center;
-        align-items: center;
-        gap: 2.4rem;
-        width: 44rem;
+    height: auto;
+    border-radius: 8px;
+    width: 44rem;
```

Comment:
How about the responsive view? can you use `max-width`?

Reply:
ah yes, theres already a max-width set for Mobile, `max-width: calc(100vw - 3.2rem);` and 44rem will fit on tablets, we want the modals to stay 44rem and in mobile they are responsive